scripts/eslint.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/bash
-
-set -e
-
-mkdir -p tmp
-
-npm run lint | tee tmp/lint-output
-
-problems=`cat tmp/lint-output | grep -oE '[0-9]* problems' | grep -o "[0-9]*"`
-ratchet="100"
```

Comment:
we now have 0 linting issues ... so now that we removed that file, the build will fail if there's even a single linting issue